isisd/isisd.h

```diff
@@ -285,10 +285,12 @@ void isis_area_add_circuit(struct isis_area *area,
 void isis_area_del_circuit(struct isis_area *area,
 			   struct isis_circuit *circuit);
 
+void delete_area_addr(void *arg);
```

Comment:
let's rename this function to be inline with the exposed function names.

Reply:
Rename `delete_area_addr()` to `isis_area_address_delete()`.